.github/workflows/rule-validate.yml

```diff
@@ -123,6 +123,15 @@ jobs:
             fi
           done
 
+#          for f in automations/*.yml
+#          do
+#            counter=$((counter + 1))
+#            yq -o=json eval "$f" >> bulk_validate_request.json
+#            if [[ $counter -ne $file_count ]]; then
+#              echo "," >> bulk_validate_request.json
+#            fi
+#          done
+
```

Comment:
```suggestion
#          for f in automations/*.yml
#          do
#            counter=$((counter + 1))
#            yq -o=json eval "$f" >> bulk_validate_request.json
#            if [[ $counter -ne $file_count ]]; then
#              echo "," >> bulk_validate_request.json
#            fi
#          done

```
is this meant to be here? this is commented out right?

Reply:
yes - it was from the `automations-expterimental` branch that we are using to test with. There's a slight issue with validation (triage vs. triage_rule) that I need to fix in the future. I was going to keep this here to remind me.

Reply:
I rm'd it for now. I'll FUP with a task.